src/backend/base/langflow/api/v1/mcp_projects.py

```
@@ -460,40 +512,108 @@
+                        if users_dir.exists():
+                            # Get the first non-system user directory
+                            user_dirs = [
+                                d
+                                for d in users_dir.iterdir()
+                                if d.is_dir() and not d.name.startswith(("Default", "Public", "All Users"))
+                            ]
+                            if user_dirs:
+                                claude_config_path = (
+                                    user_dirs[0] / "AppData" / "Roaming" / "Claude" / "claude_desktop_config.json"
+                                )
+                except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)